core/src/main/scala/org/zalando/grafter/GenericReader.scala

```diff
@@ -35,6 +35,14 @@ trait GenericReader {
   implicit def widenReader[R, A, B](r: Reader[R, A])(implicit ev: A <:< B): Reader[R, B] =
     r.map(a => ev(a))
 
+  /**
+   * compose 2 readers instances where their implicit instances are available
+   * This is particularly useful when some components define their configuration in a distinct
+   * library and you want to seamlessly extract that configuration from the application
+   * configuration. See the "In a library" page in the User Guide
+   */
+  def composeReaders[C, T, S](implicit s: Reader[T, S], t: Reader[C, T] ): Reader[C, S] =
```

Comment:
Could this also be implicit? I.e., say I have a `Reader[C,T]` and a `Reader[T,S]` in scope and want a `Reader[C,S]`. It would be nice if I could just `import GenericReader._` and have it implicitly create it.